### DISCUSSION TITLE
Hideout (DARK LEGION) [U]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#12] [Card Balance]
A card for HERETIC players.

Digital card game :
Hideout (DARK LEGION) [U]
INFLUENCE, RESTRICTED
Pay 5 [DP] to return your HERETIC to your hand.

My suggestion :
INFLUENCE RESTRICTED
Opponent may use only one tactical action per turn to attack your HERETIC.

This card needs an update. I wrote a suggestion but I'm sure we could find a better suggestion. :sunglasses:

**Comment by Kite80** [comment]
What about this: your heretics cannot be attacked as long as a non-heretic warrior is in your team

**Reply by 21-ric** [reply]
It's.....interesting. "Bio-giant" + "hideout" could be fun. :innocent:

**Reply by Kite80** [reply]
in that case they hide inside the bio giant 😇

**Comment by 21-ric** [comment]
I like @Kite80's idea : our heretics cannot be attacked as long as a non-heretic warrior is in your team.
But I don't think it's playable.

My question is : would you like to play this card?
Is it interesting to protect your heretic with "Bio-giant"(for example)?

@Sol-au-fein and @TomekPL-1 Any ideas?

**Reply by Sol-au-fein** [reply]
I dont like this idea.

**Comment by 21-ric** [comment]
Or

Your HERETIC may enter cover at any time without performing a seek cover action. 

Could works with "sewers"(in database) :

Sewers (DARK LEGION) [C]
INFLUENCE, RESTRICTED
Your Warriors may exit cover and attack during the same turn.

**Reply by Sol-au-fein** [reply]
That was one of my proposals, so i support that.
They can also have +5A from cover instead of normal +3A.
Wait, you cant exit cover and attack in same turn?

**Reply by 21-ric** [reply]
![image](https://user-images.githubusercontent.com/83040170/117487482-6ffac900-af6b-11eb-942e-0fbddcbf8ace.png)

**Reply by Sol-au-fein** [reply]
Well, in current build you can exit cover and atack in same turn. So either this rule was removed or overlooked.

**Reply by Sol-au-fein** [reply]
@framerate

**Comment by Sol-au-fein** [comment]
If rule that prohibits Exit cover and Attack will be introduced i suggest to add Sewers with original text and design Hideout to make combo with it.
